examples/quickstart.php

```diff
@@ -98,6 +100,8 @@ function (\Prooph\Common\Event\ActionEvent $actionEvent) {
                 $recordedEvent->createdAt()->format('Y-m-d H:i:s')
             );
         }
+
+        return new \Prooph\Common\Event\DefaultListenerHandler($function);
```

Comment:
Isn't this creating a new listener at each execution? Can lead to stack overflows?